README.md

```diff
@@ -5,7 +5,10 @@ Wrapper to handle importing MPAS files into xarray (https://github.com/pydata/xa
  Module does
  1. Converts MPAS "xtime" to xarray time.  Time dimension is assigned via
     `preprocess_mpas`.
- 2. Provides capability to remove redundant time entries from
+ 2. Converts MPAS "timeSinceStartOfSim" to xarray time optionally for MPAS fields coming from the
+    timeSeriesStatsAM.  Time dimension is assigned via 
+    `preprocess_mpas_timeSeriesStats`.
```

Comment:
I don't think this has to be on its own line.  I would place line 10's contents on line 9.